test/benchmarks/sails.load.test.js

```diff
@@ -205,10 +206,6 @@ function setupBenchmarks() {
  * @this {Array} benchmarks
  */
 function reportBenchmarks() {
-  var _ = require('lodash');
```

Comment:
btw, you still use lodash 3.0 which is outdated and dependencies like `lodash.isregexp`, which seems unnecessary if you already use main package.